Volume/Views/BrowserView.swift

```diff
@@ -32,8 +34,82 @@ struct BrowserView: View {
         }
     }
     
+    private var navigationTitle: String {
+        switch state {
+        case .loading:
+            return "Loading article..."
+        case .results(let article):
+            return article.publication.name
+        }
+    }
+    
+    private var isArticleLoaded: Bool {
+        switch state {
+        case .loading:
+            return true
+        case .results(_):
+            return false
+        }
+    }
```

Comment:
```suggestion
    private var isArticleLoaded: Bool { state == .loading }
```

Reply:
Tried this, it don't work :( `BrowserViewState<Results>` does not conform to `Equatable`, and I don't want to write code to try to make `Results` (which is a generic parameter) conform to `Equatable` too.

Reply:
ohhhh that makes sense